### PR TITLE
auto-cpufreq: add page

### PR DESCRIPTION
**Pull request by rafay99-epic**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/linux/auto-cpufreq.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# auto-cpufreq
+
+> Automatic CPU speed & power optimizer for Linux.
```
Comment (kbdharun) [inline pages/linux/auto-cpufreq.md:3]:
```suggestion
> Automatic CPU speed & power optimizer.
```

File: pages/linux/auto-cpufreq.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+> More information: <https://github.com/AdnanHodzic/auto-cpufreq>.
+
+- Auto-cpufreq modes and options:
```
Comment (kbdharun) [inline pages/linux/auto-cpufreq.md:6]:
```suggestion
- Run `auto-cpufreq` in a specific mode:
```

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I have some suggestions for this page.

File: pages/linux/auto-cpufreq.md (hunk 2 above)
Comment (rafay99-epic) [reply pages/linux/auto-cpufreq.md:6]: Thanks for the suggestion Brother

**Comment by rafay99-epic** [comment]
> Thanks for your contribution. I have some suggestions for this page.

Your welcome, brother, I am happy to make some changes to the page.

**Comment by rafay99-epic** [comment]
@kbdharun brother, done with all of the changes,

**Review by kbdharun — APPROVED** [review]

**Review by pixelcmtd — APPROVED** [review]

**Pull request merged**